client/src/stories/inputPassword.stories.js

```diff
@@ -0,0 +1,26 @@
+import { InputPassword } from "./inputPassword";
+import { HeaderFour } from "./inputPassword";
+import { Icon } from "./inputPassword";
+import { InputPwd } from "./inputPassword";
+
+import Lock from "../stories/assets/icon/lock.svg";
+
+export default {
+  title: "Component/InputPassword",
+  component: InputPassword,
+};
+
+export const Input_Password = () => {
+  return (
+    <>
+      <InputPassword>
+        <HeaderFour>Password</HeaderFour>
+        <>
+          <Icon src={Lock} alt="lock" />
+
+          <InputPwd type="password" placeholder="*************" />
+        </>
```

Comment:
same as above...I don't see for what you would need those empty tags...

Reply:
I'm totally with you 👍